exercises/secret-handshake/introduction.md

```diff
@@ -0,0 +1,12 @@
+# Introduction
+
+You are making a secret coding club with some friends and friends-of-friends.
+You don't know everyone in the club, so you and your friends have decided to create a secret handshake that you can use to recognise that someone is a member.
+You don't want anyone who isn't in the know to be able to crack the code.
+
+You designed the code so that it takes a binary number and turns it into a series of actions.
+
+```exercism/note
+If you aren't sure how binary works, check out [this tutorial][intro-to-binary].
```

Comment:
```suggestion
If you aren't sure what binary is or how it works, check out [this tutorial][intro-to-binary].
```

Reply:
Suggest moving this to instructions, and removing all reference to binary out of introduction.